src/views/Contacts.vue

```diff
@@ -223,29 +223,45 @@ export default {
 
 		// generate groups menu from groups store
 		groupsMenu() {
-			return this.groups.map(group => {
-				return {
-					id: group.name.replace(' ', '_'),
-					key: group.name.replace(' ', '_'),
-					router: {
-						name: 'group',
-						params: { selectedGroup: group.name },
-					},
-					text: group.name,
-					utils: {
-						counter: group.contacts.length,
-						actions: [
-							{
-								icon: 'icon-download',
-								text: 'Download',
-								action: () => this.downloadGroup(group),
-							},
-						],
-					},
-				}
-			}).sort(function(a, b) {
-				return parseInt(b.utils.counter) - parseInt(a.utils.counter)
-			})
+			const menu = this.groups.map(group => ({
+				id: group.name.replace(' ', '_'),
+				key: group.name.replace(' ', '_'),
+				router: {
+					name: 'group',
+					params: { selectedGroup: group.name },
+				},
+				text: group.name,
+				utils: {
+					counter: group.contacts.length,
+					actions: [
+						{
+							icon: 'icon-download',
+							text: 'Download',
+							action: () => this.downloadGroup(group),
+						},
+					],
+				},
+				toString: () => group.name,
+			}))
+				// Sort alphabetically
+				.sort()
```

Comment:
Not fond of this syntax with a random `.function` floating :thinking:

Reply:
Maybe in a new line with `menu.sort()` ?

Reply:
so instead of a builder-kind of construction, make menu a let variable, then start replacing it with each sort?

Reply:
You can use sort on const :)

Reply:
Wa.. i didnt remember that sort modified `this`